src/main/resources/db/changelog/changes/create-accommodation-amenities-table.yaml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
```

Comment:
We have relation many to many. This is changelog from your first PR?

Reply:
Yes, we  discussed this with you